### ISSUE TITLE
Random Session ID?

### ISSUE THREAD

**Issue by saiteja-madha** [saiteja-madha/discord-js-bot#515]
Simplify the setup of `SESSION_PASSWORD` by automating random secret generation. This eliminates user confusion and reduces security risks.

**Comment by saiteja-madha** [comment]
Since this codebase is being used by beginners, I think we can create a script to help users fill in the `.env`

**Comment by imnaiyar** [comment]
How about we use [uuid](https://www.npmjs.com/package/uuid) for that?

**Comment by imnaiyar** [comment]
Something like this would be sufficient i think
```js
const { v4 } = require('uuid')
app.use(
      session({
        secret: v4(),
        cookie: { maxAge: 336 * 60 * 60 * 1000 },
        ...// other options
    )
```

**Comment by ItzDerock** [comment]
i dont think we need to introduce a whole dependency for this, just use `node:crypto`,
```js
import { randomBytes } from "node:crypto";

session({ 
  secret: process.env.SESSION_PASSWORD ?? randomBytes(32).toString("hex"),
  ...
});
```

downside to this and the uuid approach is that upon reboot, the session secret will be different, thus invalidating all existing sessions. Prefilling the `.env` works fine for all non-docker users. 

alternatively, we can just update the readme to include instructions on how to generate a random secret, such as running `head /dev/urandom | tr -dc A-Za-z0-9 | head -c32` or link some website.

**Comment by saiteja-madha** [comment]
But won’t generating a random one and not the value be a problem?

I believe the session won’t be persisted after restart

**Comment by iamvikshan** [comment]
i would suggest having users just use their client secret (from discord)
